apps/arweave/src/ar_coordination.erl

```diff
@@ -155,15 +155,15 @@ get_self_plus_external_partitions_list() ->
 %%   {pdiff, PackingDifficulty}
 %% ]}
 get_cluster_partitions_list() ->
-	gen_server:call(?MODULE, get_cluster_partitions_list, infinity).
```

Comment:
If we go with this change (i.e. setting all the timeouts to 600000 instead of infinity), might be worth defining the timeout as a constant - probably in ar.hrl?

Reply:
I thought it might be worth putting into normal configuration?..